README.md

```diff
@@ -29,13 +29,18 @@ Gibbon is implemented in Haskell, and is set up to be built with
 [Cabal](https://cabal.readthedocs.io/en/3.4/).
 After you install Cabal, proceed to installing Gibbon's dependencies:
 
-- Ubuntu:
+- Ubuntu (works on 18.04, fails on 20.04<sup><a name="footnote1">1</a></sup>]):
```

Comment:
@adityagupta1089 It should be “up to 18.04” right?

Reply:
haven't tested on older versions, so can't say. It might work.

Reply:
Anyway, it might not be worth debating this one line in the README.

Reply:
I don’t think it’s necessary for this particular PR; testing that might take a long long time. Or maybe you can add another Github workflow that runs on 16.04 perhaps. In any case, it’s fine to merge this.